aws_lambda_powertools/tracing/tracer.py

```diff
@@ -508,7 +508,7 @@ async def async_tasks():
                 functools.partial(self.capture_method, capture_response=capture_response, capture_error=capture_error),
             )
 
-        method_name = f"{method.__name__}"
+        method_name = f"{method.__module__}.{method.__qualname__}"
```

Comment:
Add a comment above this line so we remember the expected format.

Reply:
Hi @heitorlessa! Hmm probably not a good idea. Flake mark it as possible commented code and adding an exception can make the code confusing.

```
linting::flake8..........................................................Failed
- hook id: flake8
- exit code: 1

aws_lambda_powertools/tracing/tracer.py:512:1: E800 Found commented out code
```

Instead, I updated the docstring:
![image](https://user-images.githubusercontent.com/4295173/188695388-181e9575-efcf-44ef-9b67-1cf59868d980.png)

Pls let me know what do you think about.

Reply:
Hello @heitorlessa! I added the comment above the suggested line.